src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$card-width: 200px;
+$midnight: #060b35;
+$black-shadow: #0000000d;
+$blue: #00acdc;
+$grey-blue: #616070;
+$white: #fff;
+$white-smoke: #f3f3f3;
+$dark-blue: #34568b;
```

Comment:
Variable name should describe the purpose as if the color in the palette changes to red for example you have to change the name in all places where you use it